### PR TITLE
feat(ui): Added logo to bootup screen

### PR DESCRIPTION
**Pull request by amanCypherock**

### CODE REVIEW

**Comment by rohanagarwal94** [comment]
Task linked: [PRF-6316 On device replace Cypherock text with cypherock logo](https://app.clickup.com/t/9002019994/PRF-6316)

**Comment by rohanagarwal94** [comment]
Task linked: [PRF-6316 On device replace Cypherock text with cypherock logo](https://app.clickup.com/t/9002019994/PRF-6316)

**Comment by rohanagarwal94** [comment]
Task linked: [PRF-6316 On device replace Cypherock text with cypherock logo](https://app.clickup.com/t/9002019994/PRF-6316)

**Comment by amanCypherock** [comment]
Waiting on a decision on final logo image to be used

**Review by ujjwal-cyph — CHANGES_REQUESTED** [review]
Remove the old image of logo. just to ensure it is not present in the generated binary.

File: common/interfaces/user_interface/ui_logo.c [hunk 1]
```diff
@@ -58,27 +58,19 @@
...
-  static lv_style_t style;
-  lv_style_copy(&style, lv_theme_get_mono()->style.label.prim);
-  style.text.font = &my_logo;    // This is a file in lvgl/src/lv_font
-  lv_label_set_style(label, LV_LABEL_STYLE_MAIN, &style);
+#include "events.h"
+#include "ui_multi_instruction.h"
 
-  lv_label_set_text(label, MY_LOGO_SYMBOL);
+void logo_scr_init(const uint16_t delay_in_ms) {
+  LV_IMG_DECLARE(cypherock_logo);
```
Comment (ujjwal-cyph) [inline common/interfaces/user_interface/ui_logo.c:65]: this appears to be a declaration. take out of the function.

Comment (amanCypherock) [reply common/interfaces/user_interface/ui_logo.c]: updated

**Review by ujjwal-cyph — APPROVED** [review]

**Pull request merged**